test/sysv-generator-test.py

```diff
@@ -77,7 +84,11 @@ def run_generator(self, expect_error=False):
         for service in glob(self.out_dir + '/*.service'):
             if os.path.islink(service):
                 continue
-            cp = RawConfigParser()
+            try:
+                cp = RawConfigParser(dict_type=MultiDict, strict=False)
+            except TypeError:
+                # python 2
+                cp = RawConfigParser(dict_type=MultiDict)
```

Comment:
What's the matter here? python 3 fails to initialize a RawConfigParser that's based on your subclassed dictionary unless `strict=False` is passed in? Any idea why that is the case? I guess it would be nice to add a comment at least, so the next person reading this won't have to dig for the cause.